### PR TITLE
Add "screen" in screen publication (DENNER-2609)

### PR DESCRIPTION
**Pull request by curzio-della-santa**

### CODE REVIEW

File: examples/screen-publication-with-wine.json [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+                }
+            },
+            "rating": {
+                "value": 4.3074,
+                "total": 475,
+                "valid": true,
+                "stars": 4.5
+            }
+        }
+    ],
```
Comment (ivan-wolf) [inline examples/screen-publication-with-wine.json:267]: `screen` missing here?

File: examples/screen-publications.json [hunk 2]
```diff
@@ -69,588 +157,94 @@
...
+            },
+            "promotion_type": {
+                "id": "089f8537-981a-42ba-9b0a-b3cba4a1c7d5",
+                "code": "FLUGBLATT",
+                "name": "Dennerwoche"
+            },
+            "automatic": true,
+            "article_count": 1,
+            "screen": {
+                "id": "cache-register",
```
Comment (ivan-wolf) [inline examples/screen-publications.json]: Why not `pos` (it's the acronym for Point of Sales)?

**Review by ivan-wolf — CHANGES_REQUESTED** [review]

File: examples/screen-publications.json (hunk 2 above)
Comment (curzio-della-santa) [reply examples/screen-publications.json]: ok

File: src/swagger.yml [hunk 3]
```diff
@@ -4481,6 +5142,93 @@ definitions:
...
+        $ref: '#/definitions/Country'
+      rating:
+        $ref: '#/definitions/Rating'
+
+  ScreenScreen:
+    type: object
+    properties:
+      id:
+        type: string
+        description: Unique ID
```
Comment (curzio-della-santa) [inline src/swagger.yml]: I'll update the comment  (e.g.:"tobacco" ,"pos")

File: examples/screen-publication-with-wine.json (hunk 1 above)
Comment (curzio-della-santa) [reply examples/screen-publication-with-wine.json:267]: fixed

**Pull request merged**